sotoki/sotoki.py

```diff
@@ -890,13 +882,153 @@ def get_response_headers(url):
     return "default", "default"
 
 
-def download_image(url, fullpath, convert_png=False, resize=False):
+def get_redirection_to_duplicate_or_skip(url, convert_png, resize, ext):
```

Comment:
I think we should invert the return code of this function. Would make better sense
```py
def get_identicon_shortcuts(url, convert_png, resize, ext):
    """" (skip, redirect_to) shortcuts for an identicon if possible
           
            skip (bool): whether this identicon is meaningless and should not be downloaded.
            redirect_to: URL of a copy of this identicon to redirect to """

Reply:
Done

Reply:
It's better if we call it get_image_shortcuts as this doesn't do it only for identicons

Reply:
OK then, right!